spec/Section 3 -- Type System.md

```diff
@@ -234,6 +235,31 @@ type Query {
 }
 ```
 
+While this example describes a valid GraphQL schema which happens to contain a
+type called {"Mutation"} which is not the {`mutation`} _root operation type_.
```

Comment:
This reads awkwardly, I think simply removing `While` would fix it.

Reply:
reworded

Reply:
Much better; spot on 👍